UNRELEASED_CHANGELOG.md

```diff
@@ -44,7 +44,42 @@
 
 ### ✅ Added
 - Add `ChatDomain::removeMembers` method
+- Added the following use case functions to `ChatDomain` which are supposed to replace `ChatDomain.useCases` property:
+* `ChatDomain::replayEventsForActiveChannelsCall`
```

Comment:
Is there a reason for postfixing all of these with `Call`? These methods should be similar to the ones on `ChatClient`.

Reply:
The main reasons behind the naming:
1. We would have duplicate function names in `ChatDomainImpl` class. There are already functions like `ChatDomainImp::createChannel`, or `::replayEventsForActiveChannels`. They have different return types but the same names as some of the use cases so we can't keep both of them.
2. Those existing functions (`::createChannel`, `::replayEventsForActiveChannels`) names are imperative suggesting (for me) they will perform some action while use case functions are returning a `Call` instances that needs to be executed later.

Reply:
All the existing functions are just internal API in the impl class, right? In that case we can find new names for them so that the public API is cleaner.

Reply:
Done.